packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -39,6 +39,8 @@ export interface VideoGalleryProps {
   localVideoViewOption?: VideoStreamOptions;
   /** Remote videos view options */
   remoteVideoViewOption?: VideoStreamOptions;
+  /** Limitation on number of remote video stream to display */
+  remoteVideoStreamLimitation?: number;
```

Comment:
`remoteVideoStreamLimitation` feels overly generic (limitation of what sort? Width? Count?)

How about `maxRemoteVideoStreams` or `maxRemoteVideoStreamCount`?

Reply:
Haha we think alike @prprabhu-ms 
My money is also on `maxRemoteVideoStreams`

Since all this work was done by @alcail I didn't wanna make changes unless anyone suggested them ;)

Thanks for reviewing,